include/google/google_service_root.hpp

```diff
@@ -37,9 +37,11 @@ inline void handleRootOfTrustCollectionGet(
     asyncResp->res.jsonValue["@odata.id"] = "/google/v1/RootOfTrustCollection";
     asyncResp->res.jsonValue["@odata.type"] =
         "#RootOfTrustCollection.RootOfTrustCollection";
+    constexpr std::array<std::string_view, 1> interfaces{
+        "xyz.openbmc_project.Control.Hoth"};
     redfish::collection_util::getCollectionMembers(
         asyncResp, boost::urls::url("/google/v1/RootOfTrustCollection"),
-        {"xyz.openbmc_project.Control.Hoth"}, "/xyz/openbmc_project");
+        interfaces);
```

Comment:
This seems different from the changes in https://gerrit.openbmc.org/c/openbmc/bmcweb/+/59428/15/include/google/google_service_root.hpp#48

Reply:
I will fix it, I have the same concern when making changes. 
This file is deleted in upstream bmcweb.

Reply:
We don't need to pass last parmeter by default is is set to "/xyz/openbmc_project/inventory"
Following is the getCollectionMembers function:

    getCollectionMembers(std::shared_ptr<bmcweb::AsyncResp> aResp,
                         const boost::urls::url& collectionPath,
                         std::span<const std::string_view> interfaces,
                         const char* subtree = "/xyz/openbmc_project/inventory")